src/utils/array.ts

```diff
@@ -1,5 +1,9 @@
-export async function fromAsync<T>(iterator: AsyncGenerator<T>): Promise<Array<T>> {
```

Comment:
I think this was trying to mirror [this polyfill](https://github.com/tc39/proposal-array-from-async). In a PR last week, Devin added that polyfill, then removed to avoid importing one-off libraries, and added this lil implementation. Personally I don't feel that strongly either way, but in my crystal ball I foresee this function maybe getting re-renamed back to `fromAsync`.